src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,12 +4,15 @@
  * No ending dot
```

Comment:
Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.metrics.BooleanExpressionComplexityCheck
Severity: WARN
File: /home/bjerre/workspace/violations-test/src/main/java/se/bjurr/violations/lib/example/OtherClass.java L26
 

Boolean expression complexity is 8 (max allowed is 1).

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>